Formula/g/gwyddion.rb

```diff
@@ -1,8 +1,8 @@
 class Gwyddion < Formula
   desc "Scanning Probe Microscopy visualization and analysis tool"
   homepage "http://gwyddion.net/"
-  url "http://gwyddion.net/download/2.63/gwyddion-2.63.tar.gz"
```

Comment:
agree with the change, but did not see in the news. (maybe we can followup with the livecheck per this change)